pipelines/optimus/Optimus.wdl

```diff
@@ -46,6 +46,9 @@ workflow Optimus {
   # If true produce the optional loom output
   Boolean output_loom = false
 
+  # Emptydrops lower cutoff
+  Int emptydrops_lower = 100
```

Comment:
Since this is a new parameter, make sure the documentation is updated

Reply:
I've checked and we don't document to this detail. As this is a parameter with a default value that will never be changed by the end user I also don't think it needs to be added to the default inputs file.